tests/builds/zika.t

```diff
@@ -7,7 +7,8 @@ Running from the test data directory allows us to use relative paths that won't
   $ TEST_DATA_DIR="$TESTDIR/zika"
   $ mkdir -p "$TMP/out"
   $ pushd "$TEST_DATA_DIR" > /dev/null
-  $ export AUGUR="../../../bin/augur"
+  $ export COVERAGE_FILE="$TESTDIR/../../.coverage"
+  $ export AUGUR="coverage run -a --rcfile=$TESTDIR/../../.coveragerc ../../../bin/augur"
```

Comment:
Instead of defining `AUGUR` this way with `coverage run …` in every cram test file, can we instead run cram itself under `coverage` instead in `ci.yaml`?

This would avoid the increased boilerplate in each cram test file but also mean that coverage isn't observed for local cram runs unless someone opts into it.

Reply:
Hmm. Reducing boilerplate is nice, and +1 for pulling out the coverage args into env vars set outside of cram, but my main point is to not to bake in coverage to the tests but have it be a separate concern (e.g. CI's concern).

If `AUGUR` was defined in the cram tests as:

    export AUGUR="${AUGUR:-../../bin/augur}"

then it could be overridden by `ci.yaml` to include `coverage run …` when invoking `cram`.

If you'd rather not go down this path, that's fine. This isn't a merge-blocking suggestion. In my experience, it's better to have coverage tracing happen outside of tests (i.e. the tests are unaware) rather than as integrated into the tests themselves.

Reply:
Ah, I totally agree! I just couldn't figure out how to actually accomplish that separation of concerns. That's part of why I asked for @victorlin's help here, but happy to have any other help we can get!

Reply:
Ok! Since I had already thought about this above, I went ahead and pushed a commit to separate these out. We'll see if it works…

Reply:
Looks like it worked just fine and Codecov updated the comment above with the updated coverage from the run on my change.

Reply:
Cool! Thank you for making that change, @tsibley! Do you think we can run coverage only for the cell of the test matrix that actually uploads the coverage to codecov? Or do you think there's some value in having the coverage for each cell, in case we want to upload the report as an artifact in the future?

Reply:
Ah, forgot about that detail. Not sure if worth it, but seems like not much value in running with coverage if we don't do anything with it. We can always re-enable coverage for more jobs if we decide to upload the data as a workflow artifact.

One thing that might be worth doing, particularly if we have version-dependent code paths, is to combine the coverage from all matrix jobs before upload.

Reply:
Hmm, I guess if we chose to keep running coverage on all versions, that would be fine. I don't think I would look for version-specific code coverage, though. I'd be ok with merging as is, but maybe I'll play around with the CI syntax tomorrow to get conditional coverage going.

Reply:
> I don't think I would look for version-specific code coverage, though.

Nod. It's less about looking for it explicitly and more that if we don't test version-specific code paths than coverage has some upper limit that's not 100%. :-)

Reply:
@tsibley I like the diff above since it solves #845. Maybe it'll take slightly longer to run but not a big deal.

Are there version-dependent code paths in Augur? If not, is it likely in the future?

Reply:
Not sure. Nothing obvious from a simple grep on `sys.version_info`, though that's not comprehensive. But probably not any now, and not very likely in the future (though not unreasonable)?

Reply:
@victorlin Do you want to apply the diff above before we merge? I don't feel strongly either way...

Reply:
@huddlej I'll put it in a new PR since this one looks good, and we can compare the CI differences a bit easier that way.